.github/actions/check-pr/index.ts

```diff
@@ -80,36 +80,37 @@ function getAllowedBumps(preamble: string, isBreaking: boolean): string[] {
   return [];
 }
 
-async function hasMatchingChangelog(allowedBumps: string[]): Promise<boolean> {
+async function hasMatchingChangeset(allowedBumps: string[]): Promise<boolean> {
   if (allowedBumps.length) {
     const changedFiles = getInput('changed-files').split(' ');
     const fileContents = await Promise.all(
       changedFiles.map(file => readFile(file, 'utf-8'))
     );
     return fileContents.some(fileContent =>
       allowedBumps.some(bump =>
-        new RegExp(`'@sap-cloud-sdk\/\w+': ${bump}/`).test(fileContent)
+        new RegExp(`'@sap-cloud-sdk\/.*': ${bump}`).test(fileContent)
```

Comment:
```suggestion
        new RegExp(`^'@sap-cloud-sdk\/.*?': ${bump}$`).test(fileContent)
```
[pp] Lazy quantifier and start/end line check would make this even better